src/app/slide-deck/slide-deck.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SlideDeckComponent } from './slide-deck.component';
+
+describe('SlideDeckComponent', () => {
```

Comment:
These tests likely won't even pass, I haven't bothered (and don't really see the point) of adding testing here. Just ng-cli cruft